01-Login/app.js

```diff
@@ -70,6 +70,11 @@ window.addEventListener('load', function() {
     accessToken = '';
     idToken = '';
     expiresAt = 0;
+
+    webAuth.logout({
+      returnTo: window.location.origin
```

Comment:
The unit tests use the snake_case version of this parameter:

https://github.com/auth0/auth0.js/blob/fb53e8c318416ab1b29c99907a9e3cf63fe25164/test/web-auth/redirect.test.js#L440

`returnTo` will get converted to `return_to` but, in a sample, I think it's best to use the "standard."

Reply:
Just a note for anyone looking in the future ... the SDK documentation shows `returnTo`:

https://auth0.com/docs/libraries/auth0js/v9#logout